src/plugins/usage_collection/server/routes/stats/README.md

```diff
@@ -10,7 +10,7 @@ However, the information detailed above can be extended, with the combination of
 |:----------------|:-------------:|:------------|
 |`extended`|`false`|When `true`, it adds `clusterUuid` and `usage`. The latter contains the information reported by all the Usage Collectors registered in the Kibana server. It may throw `503 Stats not ready` if any of the collectors is not fully initialized yet.|
 |`legacy`|`false`|By default, when `extended=true`, the key names of the data in `usage` are transformed into API-friendlier `snake_case` format (i.e.: `clusterUuid` is transformed to `cluster_uuid`). When this parameter is `true`, the data is returned as-is.|
```

Comment:
```suggestion
|`extended`|`false`|When `true`, it adds `clusterUuid` (Note: `usage` is never added anymore)|
|`legacy`|`false`|By default, when `extended=true`, the key names are transformed into API-friendlier `snake_case` format (i.e.: `clusterUuid` is transformed to `cluster_uuid`). When this parameter is `true`, the data is returned as-is.|
```

Reply:
Good call! IMO, I'd remove the reference to usage entirely.

```suggestion
|`extended`|`false`|When `true`, it adds `clusterUuid`.|
|`legacy`|`false`|By default, when `extended=true`, the key names are transformed into API-friendlier `snake_case` format (i.e.: `clusterUuid` is transformed to `cluster_uuid`). When this parameter is `true`, the data is returned as-is.|
```